zivid_camera/src/settings_generator.cpp

```diff
@@ -17,34 +17,27 @@
 
```

Comment:
Maybe you can add a short decription at the top what this file/program does, that it auto-generates the .cfg files and needed header utilites to handle the different setting nodes? (and what it's used for)

Reply:
This is easier to spot if one looks at the `Generator` class, but that's 500 lines into the file